lib/sqewer/connection.rb

```diff
@@ -249,4 +244,14 @@ def handle_batch_with_retries(method, batch)
 
     raise
   end
+
+  # We tried to define this list using a constant in the class, but it's not
```

Comment:
I wouldn't go that deep with code comments. Explaining this in the PR itself is good enough IMHO, since anyone interested in this sort of details can check the history 😊

Reply:
ok it makes sense for this case, I just removed it, tks!